Airport/Program.cs

```diff
@@ -0,0 +1,577 @@
+﻿
+using System;
+using System.Threading;
+using System.Text.Json;
+using System.Linq;
+using System.Xml;
+
+
+namespace Airport
+{
+    public class Program
+    {
+        readonly static string path = @"Json\";
```

Comment:
don't forget to define whether variable is private or any other, in this case should be defined as private